apps/examples/testcase/le_tc/kernel/tc_libc_unistd.c

```diff
@@ -241,6 +242,10 @@ static void tc_libc_unistd_pipe(void)
 
 	sem_wait(&pipe_sem);
 
+	/* wait for child to exit, and store child's exit status */
+	ret_chk = wait(&status);
```

Comment:
@lokeshbv Hi! Please when adding `waitpid()` call do not forget about SCHED_WAITPID configuration option: `waitpid()` API is available only when SCHED_WAITPID is enabled. Either the code logic must depend on this condition, or the dependency on SCHED_WAITPID in the le_tc/kernel/Kconfig must be added.